include/uid2/types.h

```diff
@@ -15,8 +15,8 @@ enum class IdentityScope {
 };
 
 enum class IdentityType {
-    Email = 0,
-    Phone = 1,
+    EMAIL = 0,
```

Comment:
not a breaking change?

Reply:
It is, but accepting the change for now in the interest of consistency.